src/auth/providers.js

```diff
@@ -54,6 +54,36 @@ function anonProvider(auth) {
   };
 }
 
+function customProvider(auth) {
+  const providerRoute = 'providers/custom-token';
+  const loginRoute = `${providerRoute}/login`;
+  return {
+    /**
+     * Login to a stitch application using username and password authentication
+     *
+     * @memberof userPassProvider
```

Comment:
This docstring is incorrect

Reply:
Ooops

Reply:
[nit] i think snake casing `userpass` seems weird